test/ruby_lsp_rails/code_lens_test.rb

```diff
@@ -38,17 +38,52 @@ class Test < ActiveSupport::TestCase
         assert_match("Debug", response[5].command.title)
       end
 
+      test "ignores unnamed tests (empty string)" do
+        @store.set(uri: "file:///fake.rb", source: <<~RUBY, version: 1)
+          class Test < ActiveSupport::TestCase
+            test "" do
+              # test body
+            end
+          end
+        RUBY
+
+        response = RubyLsp::Executor.new(@store, @message_queue).execute({
+          method: "textDocument/codeLens",
+          params: { textDocument: { uri: "file:///fake.rb" }, position: { line: 0, character: 0 } },
+        }).response
+
+        # The 3 responses are for the test class, none for the test declaration.
+        assert_equal(3, response.size)
```

Comment:
Excuse my ignorance on the testing API here -- is it possible to test the content of the responses to verify that they're coming from the test class? Otherwise it seems like the feature could break and the test could still potentially pass, but maybe there's no way to be more specific here.

Reply:
Good question - the other tests in this class check the response in more detail, but here the main intention was just to ensure that it doesn't raise an exception. There's a very slim chance that this single test could pass when it shouldn't, but in combination with the other tests I'm confident we'd detect the problem.